src/App.js

```diff
@@ -64,7 +86,8 @@ class App extends Component {
 
         <main>
           <p>This is a handy tool for tech recruiters who ask for fifteen years experience in technologies that have only existed for three months.</p>
-
+          
+          <p>Order by <a href="/#" onClick={this.handleSort}>Tech</a>, <a href="/#" onClick={this.handleSort}>Age</a></p>
```

Comment:
Highlighting the current active option would be nice and perhaps an icon along with it based on the direction of sorting.